stl/inc/xutility

```diff
@@ -5815,7 +5815,7 @@ namespace ranges {
                     return _RANGES next(_STD move(_First), _Result - _First_ptr);
                 }
 #else // ^^^ _USE_STD_VECTOR_ALGORITHMS / not _USE_STD_VECTOR_ALGORITHMS vvv
-                if constexpr (sizeof(_Iter_value_t<_It>) == 1 && _Is_sized /* TRANSITION, DevCom-1615707 */) {
+                if constexpr (sizeof(_Iter_value_t<_It>) == 1) {

```

Comment:
VSO-1538014 "[Feedback] ARM64 memchr implementation may not be standard conformant" is still open, so we should continue to use the `_Is_sized` guard on ARM64/ARM64EC:
```suggestion
                if constexpr (sizeof(_Iter_value_t<_It>) == 1
#if defined(_M_ARM64) || defined(_M_ARM64EC) // TRANSITION, VSO-1538014
                              && _Is_sized
#endif // defined(_M_ARM64) || defined(_M_ARM64EC)
                ) {
```